tests/integration/tier1/set-locale/use_non_english_language.py

```diff
@@ -8,8 +8,8 @@ def test_use_non_english_language(shell):
     Ref bug: https://bugzilla.redhat.com/show_bug.cgi?id=2022854
     """
     # install Chinese language pack for CentOS-8 and Oracle Linux 8
-    os = platform.platform()
-    if "centos-8" in os or "oracle-8" in os:
+
+    if "centos-8" in SYSTEM_RELEASE or "oracle-8" in SYSTEM_RELEASE:
```

Comment:
I think there was some type of justification for this, but this couldn't be?: 

```suggestion
    if SYSTEM_RELEASE in ("centos-8", "oracle-8"):
```

Reply:
@r0x0d, it should be this: https://github.com/oamg/convert2rhel/blob/main/tests/integration/tier1/set-locale/use_non_english_language.py#L10

On the rhel-8 like testing machines, we have to install Chinese language package. They are present on the rhel-7 but not  on rhel-8. However, if the code would be run also on rhel-7 then we will not encounter error in the future if the testing machine is missing the package thus making the code more robust

Reply:
> I think there was some type of justification for this, but this couldn't be?:

There actually is a justification.
We are assigning the SYSTEM_RELEASE with centos-8.4, centos-8.5 and oracle-8.7 (at the moment) it would not match.
We could do `if SYSTEM_RELEASE in ("centos-8.4", "centos-8.5", "oracle-8.7")` but then we would need to remember to change the oracle matching, whenever there is a new Oracle Linux minor released.

Reply:
Aha, makes sense! I will make a mental note about this the next time I review any integration tests. Thanks guys!